packages/stack/blockchain-client/package.json

```
@@ -4,7 +4,7 @@
   "author": "Iuri Matias <iuri.matias@gmail.com>",
   "contributors": [],
   "description": "Provides ability to register a blockchain technology in Embark, ie Ethereum.",
-  "homepage": "https://github.com/embarklabs/embark/tree/master/packages/blockchain-client/proxy#readme",
+  "homepage": "https://github.com/embarklabs/embark/blob/master/packages/stack/blockchain-client#readme",
```

When I loaded the revised URL it got redirected to:

```suggestion
  "homepage": "https://github.com/embarklabs/embark/tree/master/packages/stack/blockchain-client#readme",
```